<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addModule</class>
 <widget class="QMainWindow" name="addModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>221</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(46, 52, 54);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>205</width>
      <height>129</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">color: rgb(238, 238, 236);
font: 10pt &quot;Courier 10 Pitch&quot;;
alternate-background-color: rgb(136, 138, 133);
border-right-color: rgb(0, 0, 0);
border-bottom-color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="vModuleID">
         <property name="styleSheet">
          <string notr="true">color: rgb(238, 238, 236);
font: 10pt &quot;Courier 10 Pitch&quot;;
alternate-background-color: rgb(136, 138, 133);
border-right-color: rgb(0, 0, 0);
border-bottom-color: rgb(0, 0, 0);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">color: rgb(238, 238, 236);
font: 10pt &quot;Courier 10 Pitch&quot;;
alternate-background-color: rgb(136, 138, 133);
border-right-color: rgb(0, 0, 0);
border-bottom-color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="vModuleName">
         <property name="styleSheet">
          <string notr="true">color: rgb(238, 238, 236);
font: 10pt &quot;Courier 10 Pitch&quot;;
alternate-background-color: rgb(136, 138, 133);
border-right-color: rgb(0, 0, 0);
border-bottom-color: rgb(0, 0, 0);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">color: rgb(238, 238, 236);
font: 10pt &quot;Courier 10 Pitch&quot;;
alternate-background-color: rgb(136, 138, 133);
border-right-color: rgb(0, 0, 0);
border-bottom-color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Classifier</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="vModuleClassifier">
         <property name="styleSheet">
          <string notr="true">color: rgb(238, 238, 236);
font: 10pt &quot;Courier 10 Pitch&quot;;
alternate-background-color: rgb(136, 138, 133);
border-right-color: rgb(0, 0, 0);
border-bottom-color: rgb(0, 0, 0);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pbOk">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 87, 83);
color: rgb(238, 238, 236);
font: 14pt &quot;Courier 10 Pitch&quot;;
alternate-background-color: rgb(136, 138, 133);
border-right-color: rgb(0, 0, 0);
border-bottom-color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
